composer.json

```diff
@@ -50,7 +50,7 @@
         "laminas/laminas-i18n": "^2.11.0",
         "laminas/laminas-inputfilter": "^2.12.0",
         "laminas/laminas-json": "^3.2.0",
-        "laminas/laminas-log": "^2.13.0",
+        "laminas/laminas-log": "^2.0.0",
```

Comment:
@Koen1999 
Are you sure with this change? See: https://github.com/laminas/laminas-log/releases/tag/2.15.0

Reply:
Right, but is the change then really necessary?

Reply:
Nope, it's not. It probably was changed by composer while updating some other package. But I think I'll change them anyway to help composer a bit. Maybe it speeds it up a bit because there's less versions to choose from.

Reply:
The change is just confusing. 😉

Reply:
The whole project is sometimes xD